Description: Upstream changes introduced in version 2.4.0-8
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libfcgi (2.4.0-8) unstable; urgency=low
 .
   * Switch to dpkg-source 3.0 (quilt) format
 .
 The person named in the Author field signed this changelog entry.
Author: Tatsuki Sugiura <sugi@nemui.org>
Origin: vendor, http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=504963
Bug-Debian: http://bugs.debian.org/504963
Forwarded: no
Last-Update: 2010-02-08

Index: libfcgi/libfcgi/fcgio.cpp
===================================================================
--- libfcgi.orig/libfcgi/fcgio.cpp	2010-01-23 19:13:53.000000000 +0900
+++ libfcgi/libfcgi/fcgio.cpp	2010-01-23 19:13:55.000000000 +0900
@@ -22,6 +22,7 @@
 #define DLLAPI  __declspec(dllexport)
 #endif
 
+#include <cstdio>
 #include <limits.h>
 #include "fcgio.h"
 
